php/src/vaas/Vaas.php

```diff
@@ -146,22 +139,12 @@ public function forFileAsync(string $path, ?ForFileOptions $options = null, ?Can
                 throw new VaasClientException('File does not exist');
             }
 
-            if ($options === null) {
-                $options = new ForFileOptions(
-                    [
-                        'vaasRequestId' => null,
-                        'useHashLookup' => $this->options->useHashLookup ?? true,
-                        'useCache' => $this->options->useCache ?? true,
-                    ]
-                );
-            }
+            $options = $options ?? ForFileOptions::default();
```

Comment:
The same here like in line 84.
```suggestion
            if ($options === null) {
                $options = new ForFileOptions($this->options->useCache, $this->options->useHashLookup, null);
            }
```